app/controllers/sidebar_sections_controller.rb

```diff
@@ -13,6 +22,7 @@ def create
 
     if sidebar_section.public?
       StaffActionLogger.new(current_user).log_create_public_sidebar_section(sidebar_section)
+      MessageBus.publish("/refresh-sidebar-sections", nil)
```

Comment:
I think for all the publishes you can pass `group_ids` as the argument after nil, and just give it the `SiteSetting.enable_custom_sidebar_sections_map` which will be the group IDs of people using this at the moment, that way we don't have to publish to everyone in message bus.

I guess when that is removed though there are no `user_ids` we can use, since everyone will see this.

Also I think we should probably pass the sidebar sections JSON in the payload, not `nil`, that way we don't trigger a bunch of AJAX requests for all active users to `/sidebar_sections.json`. Since it's public sections, we can just pass the array of public sections here, which is not a per-user thing.